.github/workflows/development.yml

```diff
@@ -13,10 +13,13 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
+    - name: Checkout code
+      uses: actions/checkout@v2
     - name: install docker-compose
       run: pip install --no-cache-dir docker-compose
```

Comment:
I'm not sure if this is what's causing the CI issues, but I don't think you should need to install docker-compose.

Reply:
The CI issue was being caused by the filename of the dockerfile. Now the CI fails because it could not find the [credentials in the asset folder](https://github.com/Health-Union/snowshu/pull/57/checks?check_run_id=2141599100#step:5:812).

Reply:
Oof. Might be worth letting the CI tests fail for this PR, and we can decouple the unit tests from the integration test assets in another one.